types_test.go

```diff
@@ -0,0 +1,62 @@
+package surrealdb_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"testing"
+
+	"github.com/surrealdb/surrealdb.go"
+	"github.com/test-go/testify/suite"
```

Comment:
If we could not use testify that would be great, Go has a fantastic standard library that does most of the work for us, and testify is a huge dependency (go has no concept of dev dependencies, so the entire testify and its many dependencies are pulled if you want to use this driver)